build/package/Dockerfile.helm

```diff
@@ -26,27 +18,17 @@ ENV GOBIN /usr/local/bin
 RUN chmod g+w /go
 
 # Install Helm.
-RUN mkdir -p $HELM_PLUGINS \
-    && mkdir -p /tmp/helm \
+RUN mkdir -p /tmp/helm \
     && cd /tmp \
     && curl -LO https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \
     && tar -zxvf helm-v${HELM_VERSION}-linux-amd64.tar.gz -C /tmp/helm \
     && mv /tmp/helm/linux-amd64/helm /usr/local/bin/helm \
     && chmod a+x /usr/local/bin/helm \
     && helm version \
     && helm env \
-    && HELM_DATA_HOME=${HELM_PLUGINS%/*} helm plugin install https://github.com/databus23/helm-diff --version v${HELM_PLUGIN_DIFF_VERSION} \
-    && HELM_DATA_HOME=${HELM_PLUGINS%/*} helm plugin install https://github.com/jkroepke/helm-secrets --version v${HELM_PLUGIN_SECRETS_VERSION} \
-    && sops --version \
-    && ls -lah $HELM_PLUGINS \
+    # && sops --version \
```

Comment:
@michaelsauter What is `sops` for?

Reply:
It's used by the `helm-secrets` plugin and can be used to store secrets encrypted in the repo.

Reply:
Given our conversation earlier, once this line is moved to the plugin installation further down, we can merge!